modin/pandas/__init__.py

```diff
@@ -337,6 +340,7 @@ def init_remote_ray(partition):
     "datetime",
     "NamedAgg",
     "api",
+    "read_xml",
```

Comment:
Why wasn't it provided in API before?

Reply:
Well, I would rather not... :) call me lazy, but a 1-line change where I touch a lot of things here and there could go along with others IMHO.

Also, it's not as serious as you'd think, the only case it would break before would be
```python
from modin.pandas import *
df = read_xml('foo.xml')
```

Note that we _don't_ recommend star-imports, and it won't impact the usual `import modin.pandas as pd` case.